chromium_src/base/check.cc

```diff
@@ -0,0 +1,15 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
```

Comment:
I'm not sure why anything NACL related would be showing up in the ios build. Are we sure this is correct? https://github.com/brave/brave-browser/blob/master/lib/config.js#L306 ?
Does it build if you remove `delete args.enable_nacl`?